crates/runtime/src/model.rs

```diff
@@ -14,69 +15,66 @@ pub struct Model {
 pub type Result<T, E = Error> = std::result::Result<T, E>;
 #[derive(Debug, Snafu)]
 pub enum Error {
-    #[snafu(display("Unknown data source: {model_source}"))]
-    UnknownDataSource { model_source: String },
+    UnknownModelSource {
```

Comment:
You need to add a display message with the source error specifically printed otherwise the source error will not be printed by default.